shaggy-page-action/popup.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  </head>
```

Comment:
I think you should declare the charset here or you will get an error in the console.